tests/fixtures/fields_data_2.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Consider renaming this to "fields_data_invalid.json"? Also given that we're saving things in yml format, does having things in `.json` matter?

Reply:
This tests an internal function directly, so it does not involve YAML.